utils/synchronize.ts

```diff
@@ -0,0 +1,19 @@
+import { writeFile } from 'fs/promises';
+import { Client } from 'pg';
+import { queryDatabase } from '../routes/databaseFunctions';
+import path from 'path';
+
+const FILE_PATH = path.resolve(__dirname, 'data.json');
+
+const synchronizeLocal = async (client: Client) => {
```

Comment:
this is what i mean - the function is declared as async. i guess you don't HAVE to await it, but you should